sjb/config/test_cases/test_branch_origin_extended_conformance_install_system_containers.yml

```diff
@@ -95,12 +104,18 @@ extensions:
       script: |-
         docker_image_tag="$( cat ./ORIGIN_TAG )"
         docker_repository="${OS_IMAGE_PREFIX:-"openshift/origin"}"
-        docker inspect "${docker_repository}:${docker_image_tag}"
+        [[ "$(sudo atomic containers list -n --no-trunc -f runtime=runc -f image=${docker_repository}:${docker_image_tag} --json | jq '.[0].running')" == "true" ]]
```

Comment:
@ingvagabund [here](https://github.com/ashcrow/atomic/commit/4227d772c04daa46de0eaf6be524167755683cec) is a basic jsonpath implementation with the ``atomic containers list`` command. It adds an option called ``--jsonpath=`` if ``jsonpath_rw`` is available on the system. Feel free to base whatever changes you need on that commit.

```
# atomic containers list --jsonpath='[*].id'
[
    1,
    2
]
```